ui/components/app/beta-header/index.js

```diff
@@ -50,7 +51,7 @@ const BetaHeader = () => {
         }}
         aria-label={t('close')}
       >
-        <i className="fa fa-times" />
+        <Icon name={ICON_NAMES.CLOSE} />
```

Comment:
Can you please add screenshots like the below at each instance so we can easily review visual changes? It takes more time on the author side but will save a lot of time and possible mistakes on the review side. You could use local storybook and the [Github page](https://metamask.github.io/metamask-storybook/?path=/story/components-app-betaheader--default-story) to have each side by side so you don't have to rebuild both

Or a production build in Firefox and your local build in Chrome would also work if there are no stories documenting the icon updates.

### Before

<img width="725" alt="Screenshot 2023-02-06 at 3 01 20 PM" src="https://user-images.githubusercontent.com/8112138/217107048-f64c7b1b-e266-4511-a516-ca0b9d26e73a.png">

### After

<img width="724" alt="Screenshot 2023-02-06 at 3 01 24 PM" src="https://user-images.githubusercontent.com/8112138/217107079-a5f668f8-6e86-4ff8-a00f-15f6808f2b25.png">


Some you will need to add `color` or `size` props to. I think `color={IconColor.warningInverse}` would work here

```suggestion
        <Icon name={ICON_NAMES.CLOSE} color={IconColor.warningInverse />
```